### ISSUE TITLE
Channel priority should be set to strict and conda info should be printed with conda-store

### ISSUE THREAD

**Issue by costrouc** [conda-incubator/conda-store#359]
Right now some solves are pulling in multiple channels that were not specified.

**Comment by costrouc** [comment]
I think that channel priority strict is already set. It would be nice to have the output of:
 - `conda info`
 - `conda config --show`

**Comment by kcpevey** [comment]
I'd also like to confirm that we are setting channel priority to strict and ensure that its not pulling in things from defaults or anywhere else.

**Comment by nkaretnikov** [comment]
Note: the block of code that Chris refers to above is actually this (no hash was specified in the link and it's been refactored since then):

`git show 5f7adfb5737edef307f0a70a42f7d04d66adce30:conda-store-server/conda_store_server/build.py`:
```py
145 def build_environment(conda_command, environment_filename, conda_prefix):
146
147     return subprocess.check_output(
148         [
149             conda_command,
150             "env",
151             "create",
152             "-p",
153             conda_prefix,
154             "-f",
155             str(environment_filename),
156         ],
157         stderr=subprocess.STDOUT,
158         encoding="utf-8",
159     )
```

**Comment by nkaretnikov** [comment]
Status update: this should be fixed by #693.
